### ISSUE TITLE
Bullets are incorrectly showing up on `li.task-list-item`

### ISSUE THREAD

**Issue by revin** [npm/newww#1967]
TL;DR we need `.task-list-item { list-style-type: none; }` on npmjs.com

Hi all, now that npmjs.com supports GitHub-style task lists, a package like [`package-skeleton`](https://www.npmjs.com/package/package-skeleton) has checkboxes:

<img width="585" alt="screen shot 2016-07-18 at 10 49 00 am" src="https://cloud.githubusercontent.com/assets/94850/16924671/5046d0ca-4cd5-11e6-9c02-c5c2c68cad00.png">

but the same markdown on GH doesn't have bullets (from [`amercier/npm-package-skeleton`](https://github.com/amercier/npm-package-skeleton)

<img width="610" alt="screen shot 2016-07-18 at 10 49 11 am" src="https://cloud.githubusercontent.com/assets/94850/16924713/86455002-4cd5-11e6-8a80-00bc89a2656c.png">

GH's stylesheets have `.task-list-item { list-style-type: none; }` so I imagine we want the same or similar?
